airbyte-bootloader/src/main/resources/application.yml

```diff
@@ -36,6 +36,9 @@ airbyte:
         address: ${VAULT_ADDRESS:}
         prefix: ${VAULT_PREFIX:}
         token: ${VAULT_AUTH_TOKEN:}
+  worker:
```

Comment:
May have missed this in the original PR, why do we have worker config in bootloader?